src/client/components/pages/about.js

```diff
@@ -15,11 +15,16 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
+/* eslint valid-jsdoc: ["error", { "requireReturn": false }] */
```

Comment:
Let's update the rules globally for this, since we seem to be making exceptions everywhere for it. It seems like the right thing to be doing.